### ISSUE TITLE
Mount 的時候出現 returned non-zero exit status 32

### ISSUE THREAD

**Issue by minshenglin** [minshenglin/keepalived_rbd_nfs#2]
Log:

```
[2017-10-17 13:26:13][INFO ] Change to master state...
[2017-10-17 13:26:13][INFO ] Starting to enable repos...
[2017-10-17 13:26:13][INFO ] Map RBD image hasql24oldbak2
[2017-10-17 13:26:13][INFO ] RBD image hasql24oldbak2 is mapped on /dev/rbd1
[2017-10-17 13:26:13][INFO ] Mount /dev/rbd1 to /mnt/hasql24oldbak2
[2017-10-17 13:26:14][ERROR] Mount to /mnt/hasql24oldbak2 failed: Command '['/bin/mount', '/dev/rbd1', '/mnt/hasql24oldbak2']' returned non-zero exit status 32
```

應該是 RBD Image 尚未格式化:
```
[vagrant@client-1 ~]$ sudo mount /dev/rbd0 /mnt
mount: /dev/rbd0 is write-protected, mounting read-only
mount: unknown filesystem type '(null)'
[vagrant@client-1 ~]$ echo $?
32
```

**Comment by minshenglin** [comment]
Fixes by c97d965d4a67dd5a47e52b05f3026a682c82bca3

**Comment by minshenglin** [comment]
用 systemctl restart keepalived 測試, 
backup 狀態會等待 stop 結束後才執行
